### ISSUE TITLE
`-v` should be passed to WSL when it is passed on host

### ISSUE THREAD

**Issue by RuiNtD** [topgrade-rs/topgrade#521]
## I want to suggest some general feature
Topgrade should pass `-v` / `--verbose` to the WSL step when it's passed to the main `topgrade` command.

**Comment by SteveLauC** [comment]
Rationale for this change?

**Comment by RuiNtD** [comment]
Sorry, let me rephrase: `-v` should be passed to WSL when `-v` is passed to the main command.

When using `-v` on Windows, it stops being Verbose during the WSL step.

**Comment by SteveLauC** [comment]
Get it, this will be a nice improvement!

**Comment by ssmendon** [comment]
I submitted #556 for this, by the way. I tested it out on Windows, and it seemed to give me the expected result.

I'd appreciate any extra testing for it though.

PS. Could someone test the `-y` option, whether that's passed to WSL? It didn't look like it's passed to the `topgrade` executable in WSL based on how I tried to add the `-v` flag.

@RuiNtD

**Comment by SteveLauC** [comment]
> I submitted https://github.com/topgrade-rs/topgrade/pull/556 for this, by the way. I tested it out on Windows, and it seemed to give me the expected result.

Thanks for working on it!

> Could someone test the -y option, whether that's passed to WSL? It didn't look like it's passed to the topgrade executable in WSL based on what I first tried when adding the -v flag.

The verbose argument would be useful for debugging, say a user is having issues with WSL, it can paste the output of `topgrade -v` from its host without having to execute it inside of the WSL.

For the `assume_yes` (`-y`) option, I haven't seen much value in it, users can configure this option in the configuration file if they want
